### ISSUE TITLE
Adding unpaid opportunities

### ISSUE THREAD

**Issue by karllhughes** [malgamves/CommunityWriterPrograms#11]
Maybe this goes against the goal of the project, but do we have any interest including unpaid opportunities that are still a good way to gain exposure?

For example, A List Apart and DZone are both high-quality publications that attract big readership, but they're not paid opportunities.

I'm fine leaving them out, but we could also create another section for unpaid opportunities if we wanted?

**Comment by malgamves** [comment]
Yeah, I've been informed of a program already on the list that isn't paid anymore - been looking for a place for that to go. We could have a section after, or another README file? I'm not sure which one is better. A resourse for publications that can help with reach alone is also very useful.

**Comment by karllhughes** [comment]
I like having a second section in the same file for now. Maybe when this list gets too big we can break it up into multiple files, but it's not too bad right now.

**Comment by malgamves** [comment]
That makes sense! Worth thinking about later.

**Comment by hjdfdr44** [comment]
I need help to use the program for my subject of [Reasons to join Illuminati Illuminati](https://jointtheilluminati.com/) which is hosted on a dynamic page.
